server/api/smallgroup/index.js

```diff
@@ -12,8 +12,8 @@ router.get('/', auth.hasRole('mentor'), controller.index);
 router.post('/', auth.hasRole('mentor'), controller.create);
 
 // Generate a daycode or return the current day code for your smallgroup
-router.post('/daycode', auth.hasRole('mentor'), controller.daycode);
-router.delete('/day/:dayCode', auth.hasRole('mentor'), controller.deleteDay);
+router.post('/:id/daycode', auth.hasRole('mentor'), controller.daycode);
```

Comment:
Why do this? The factory can set the id parameter to be daycode and use the original endpoint.  Mentors should only be generating daycodes for their own small group anyways.